article-analyzer/src/main/java/co/inventorsoft/academy/spring/articleanalyzer/notifier/EmailNotificationService.java

```diff
@@ -0,0 +1,19 @@
+package co.inventorsoft.academy.spring.articleanalyzer.notifier;
+
+import co.inventorsoft.academy.spring.articleanalyzer.model.User;
+import co.inventorsoft.academy.spring.articleanalyzer.repository.UserRepository;
+import com.google.gson.Gson;
+import org.springframework.context.ApplicationContext;
+import org.springframework.stereotype.Service;
+
+@Service("emailNotificationService")
```

Comment:
this name is needless, spring does create the bean with the same name as is specified here